ml/rl/thrift_handler.py

```diff
@@ -0,0 +1,41 @@
+import importlib
+import os
+
+if importlib.util.find_spec("thriftpy") is not None:
```

Comment:
I chose this way because that way could execute code (see https://stackoverflow.com/questions/14050281/how-to-check-if-a-python-module-exists-without-importing-it ).